spec/features/step_by_step_actions_spec.rb

```diff
@@ -58,6 +88,10 @@ def action_html
   end
 
   def and_I_am_the_step_by_step_author
-    @current_user = @review_requester
+    login_as @review_requester
+  end
+
+  def and_I_am_the_reviewer
+    @current_user = @reviewer
```

Comment:
```suggestion
    login_as @reviewer
```

Reply:
Updated as suggested, but required the `can_submit_for_2i` to be moved in the `StepNavActionsHelper` to be able to pass the `current_user` in a similar fashion.